File: Latihan A/Product.java
//kelas yang digunakan untuk mengimplementasikan kelas Product
public class Product{

	//atribut private
	private int price = 0;
	private String idProduct = "";

	//konstruktor
	public Product(int price, String idProduct){
		this.price = price;
		this.idProduct = idProduct;
	}

	public Product(){

	}

	//getter dan setter price
	public void setPrice(int price){
		this.price = price;
	}

	public int getPrice(){
		return this.price;
	}

	//getter dan setter idProduct
	public void setIdProduct(String idProduct){
		this.idProduct = idProduct;
	}

	public String getIdProduct(){
		return this.idProduct;
	}

}

File: Latihan B/Python/Product.py
#kelas yang digunakan untuk mengimplementasikan kelas Product
class Product:

	#konstruktor
	def __init__(self):
		self.price = 0
		self.idProduct = ""


	#getter dan setter untuk atribut price
	def setPrice(self, price):
		self.price = price

	def getPrice(self):
		return self.price

	#getter dan setter untuk atribut idProduct
	def setIdProduct(self, idProduct):
		self.idProduct = idProduct

	def getIdProduct(self):
		return self.idProduct